tests/baselines/reference/arrayAssignmentTest1.errors.txt

```diff
@@ -26,10 +26,13 @@ tests/cases/compiler/arrayAssignmentTest1.ts(70,1): error TS2322: Type 'C3[]' is
     Property 'C2M1' is missing in type 'C3'.
 tests/cases/compiler/arrayAssignmentTest1.ts(75,1): error TS2322: Type 'C2[]' is not assignable to type 'C3[]'.
   Type 'C2' is not assignable to type 'C3'.
+    Property 'CM3M1' is missing in type 'C2'.

```

Comment:
The primitive baselines seem good to simplify, but what about these? Is it OK to drop these elaborations?